assets/js/components/DataBlockGroup.js

```diff
@@ -39,12 +39,11 @@ export default function DataBlockGroup( { className, children } ) {
 			return;
 		}
 
-		// Reset font sizes first to get accurate measurement, specifically on resize.
```

Comment:
@ankitrox, sorry, one last thing - please can you restore this comment, and the position of the call to `setFontSizes()` to minimise the changes in this PR?